jobtracker.go

```diff
@@ -42,8 +42,12 @@ const (
 )
 
 var countersToKeep = map[string]string{
-	"FileSystemCounter.HDFS_BYTES_READ":    "hdfs.bytes_read",
-	"FileSystemCounter.HDFS_BYTES_WRITTEN": "hdfs.bytes_written",
+	"FileSystemCounter.HDFS_BYTES_READ":    "bytes_read",
+	"FileSystemCounter.S3_BYTES_READ":      "bytes_read",
+	"FileSystemCounter.FILE_BYTES_READ":	"bytes_read",
+	"FileSystemCounter.HDFS_BYTES_WRITTEN": "bytes_written",
+	"FileSystemCounter.S3_BYTES_WRITTEN":   "bytes_written",
+	"FileSystemCounter.FILE_BYTES_WRITTEN": "bytes_written",
```

Comment:
If we send the counters separately we can display the summed value and put the separate pieces in the `title` so it's available on hover.